src/Pages/HomeComponents/ProjectsList.jsx

```diff
@@ -0,0 +1,25 @@
3zHvV/JSyMcRGy46HHRcTu9y8CgkuJJAOY5ry02uR9bIdl1mszP1cr2LyCl4yFofcEX0D0gdHgsDSgE6bje5OYzs2/Qtoqn80NabOkOg0jouDd3gAT4LXMdvBK3RA4osAa3UBoi1r+4+K9ki2tCyTW4rTNjAu17svquZme6y8xwOLmt9UvF26XNuNuaxH6Qdsb8d681FPMbPex5DgC1wBLTssQoXM1087eRvVPCGNDR0dO3aV6lkDWucdTQSfBUcNa8QxiS+noDSvr7j2qhj0ujA/8Ais33nP4XVk5bMHLgjq0o7TjFb7GqzzF7nPdrcbncraeJ2kARbmh3gcwr/DKTjpGs+rredgGvd4q64Rx2mBAteNvwuPyXE6tum6j4nb6xKaprh5cC/wCCzbRyH+MD3D/1ZWqbdjwOlrvksfwbbaEna93yAVzS14c4se3QfqsTcHsXWw2VKPgcfFSXWyT3ssMHltJo9Dh8RmPzVDhH7Zv3bfxOVWKPQqA3Y6w7jq+BVLhH7Vv3bfxOVOL/ANL8UedGfNbhcxSIi5J2wtXrBeST7bvmVtC1ir9pJ9t3zKjIto6s6FwKFqKMfxS/jK1fFPbz/fSfiK2ngWf1OP7Un4ytWxT28/3sn4ivtegn8P8A1R8p0v8AO/3P8lqiIvoDjBERAYfEPaO8PkFd8GP+bSffx/NWmIe0d4fIK64Nn9cpfvmfNRqfJLwf2LYfMvodrRWjZiO1VhM2xJNrZm+Vl8wpJnXcWiqixjMXic8MbpG5DQ6w0bnxuskvU09CCaehKKEXp6SihEBKhSiAhWtQOcrtUKgaj4KM1kSi8ygSrT9Jwa+Mbbxt8ldLWamAtdJGdpLe0HMLDiKsqaTSubKFKNRtNlyzEowYSXXEbZxYXvkQGe9oXnFGU87bmdglGo3cWAfu22duv5LDosPt0uC5l/u6m1Zt8v4PRj4w6M0jAWjmTXBB/hdbM95FwtiwTDDADpSOcT9UEiMeB1ntWDooDJI1o6Tn2Aa1uBIGfQM1rwtR1U5NWMlbCU6Mlstt/wB4ELC8KJQGRsuLucXAXzOiLH8Su/03Tdcz+7ctK9IMUc0tHW0rzNLTSxh8LX20ow7TJbfK9xY7Q7sW6WDqVU4NNX32ZXDEwpyU7p270bngdFxUd3Dnvs53YOgf7tVlwpi9k/7TT8x+auY+ENM5gfplpLdLi3NIkH8JAuL+K1rhJi73RPfFoyyNLTHFcgesAfgSfBUVMFUdLqowfdk+ZKONhGqpyev2M5hOINjha3RLjdxOYAzK81lUJCHBha4fWDr3+C1bA8ce5rWVMEkL8+eADEfjcf7mr/8ATdN1rf6X7lNYWvCKi4vJcGYMRiOsnJ3ybZnzVNc+KQ5EW4zLLLUQrTHJmvkBYbjQAvYjpdtVnSV8UpIieHltibAi19WsJU6/BY8cpRpuMlbNGzoyo3WtxT+xSREXIPoAtYq/aSfbd8ytnWsVntJPtu+ZUZFtHVmzYASIGEXBu7UbdJWPqjeSQnMl7s/Eq+wL2De9/wAyrGp9d/23fMr7PoDR/tR8X0l/vn+5/coopRfRHPIRSoQMw+Ie0d4fIK64Of8AMpfvmfNWuIe0d4fIKrgry2ogcNYkaR71Gr/rl4P7FsNV9DsSweJ1umdFp5g/uO3uUy4m5zNG1nHK41WVzQ4bo2dJYnob0Dv2lfH3urI6cpOr8MNN56wWjsRJJkfqA9HaVnVZXVeCS+R8FdTaWRZ1eysiuihSrSJCKVCAIiIChUVcUdhJJHHfUHyNbfuuqMmJ0xBHKIPOj3q4lp2PtpsY62rSaDb3rzyKLqovLbuTIGO/SdP18HnR71ia7GqCT9uwOb6rw5h/PMLZ+RRdVF5bdycji6qPy27lnqUNpWv5q/5RfCtsu9udvwaBNX05zE0V/vGWPxVIVsPXRDt4xu9dE5HF1Uflt3JyOLqo/LbuWN9GR7XL1NXvB9nmarh2MUMIynYXH1nEs35BZabE6fQd9PB6p/bR7O9ZTkUXVR+W3cnIouqi8tu5bKVDYVr5eFvyzLUrbbvbn6HIxVxdZH5jd6nlcXWR+Y3eut8ji6qPy27k5HF1Uflt3Lue832efocv2Ndrkck5XF1kfmN3pyuLrI/MbvXW+RxdVH5bdycji6qPy27l57yfZ5+g9jXHkck5XF1kfmN3rAcoj/fZ/W1d65HF1Uflt3JyOLqo/LbuT3k+zz9CSwiW/kcj4JVUTXTXkjF2s1yNHSe1Zyorob+1i1dazt7V0DkcXVR+W3cnIouqi8tu5cjHU1ipOTdr279PI14X/HntLM51y2HrYvNZvTlsPWxeazeui8ji6qPy27k5HF1Uflt3Lne649t+XqdH3jLs8znPLYeti81m9a5V1MfGSfSR+u79o3au08ji6qPy27k5HF1Uflt3I+iov9fL1Jw6TlH9PP0OcYHWxCAXliGbtcrB0ntWPqKuLTf9JH6zv2jdp7V1fkcXVR+W3cnI4uqj8tu5dfAy9k0+LJLgcbE0uum53tdt+ZyTlUfWR+Y3enK4usj8xu9db5HF1Uflt3JyOLqo/LbuXR95Ps8/Qz+xrjyOScri6yPzG705XF1kfmN3rrfI4uqj8tu5ORxdVH5bdye8n2efoe+xrtcjh2IVMfGO57Oj67di9YVUxieG72D6RuZe3au38ji6qPy27k5HF1Uflt3KM+kHKLjs6rj6Hqwvf/fM0bl0PXQ+azeshSVMEoOnVMGjYW4+O1ujWVtPI4uqj8tu5ORRdVF5bdy5CoriXRpJPPMwlK+ljN21ERPbUR29wKuhicHXwedHvWR5FF1UXlt3JyKLqovLbuUlTSL00lZIosxWnI9vB50e9XMMzXgOY5r2nU5rg4HxCp8ih6qLy27lVjja0WaA0bGgAe4Kwie0REBKhSiAIiIAihEBKhEQBSoUoCERCgClFCAIiIAilQgCKUQBQiICUREBCIiAIpUIApUKUARQpQEIpUIAiIgJUKVCA//Z"
```

Comment:
It might be better to import the image from an asset directory so that you can keep your code more readable :)

Reply:
Okay great! If you get stuck on it later, drop a message into the support channel and we'll see if we can sort it out :)